targets/targets.json

```diff
@@ -1111,9 +1111,8 @@
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM"],
         "program_cycle_s": 2,
         "extra_labels": ["STM", "STM32F4", "STM32F407", "STM32F407xG", "STM32F407VG"],
-        "macros": ["LSI_VALUE=32000"],
+        "macros": ["LSI_VALUE=32000", "TRANSACTION_QUEUE_SIZE_SPI=2"],
```

Comment:
Also after a closer look where LSI_VALUE is defined/used, it seems like the entry in targets.json is redundant as STM32F4 HAL already defines 32000 by default, which means it can be removed.

https://github.com/ARMmbed/mbed-os/blob/9b8efb2659540654946a79b08916f941c02a14d0/targets/TARGET_STM/TARGET_STM32F4/device/stm32f4xx_hal_conf.h#L125-L127